pkg/destroy/vsphere/vsphere.go

```diff
@@ -21,6 +23,8 @@ import (
 	"github.com/openshift/installer/pkg/destroy/providers"
 	installertypes "github.com/openshift/installer/pkg/types"
 	vspheretypes "github.com/openshift/installer/pkg/types/vsphere"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
Move this up into the section above the installer imports, with the govmomi imports (which should really be in the same section as the logrus imports).